tendrl-dashboard.spec

```diff
@@ -8,16 +8,57 @@ Group:   Applications/System
 Source0: %{name}-%{version}.tar.gz
 URL: https://github.com/Tendrl/dashboard
 
-Requires: tendrl-api
+BuildRequires: npm
+BuildRequires: nodejs-packaging
+BuildRequires: fontconfig
 
 %description
 Contains the JavaScript GUI content for the Tendrl front-end components
 (dashboard, login screens, administration screens)
 
 %prep
-%autosetup
+%setup -q -n %{name}-%{version}
+%nodejs_fixdep foomodule
```

Comment:
"foomodule" is just an example in Fedora's wiki. It needs to be the name of a real package here.